aiounifi/models/device.py

```diff
@@ -330,6 +330,16 @@ class TypedDeviceUplink(TypedDict):
     uplink_remote_port: int
 
 
+class TypedDeviceUpdtimeStats(TypedDict):
+    """Device uptime stats type definition."""
+
+    wan: TypedDeviceUpdtimeStatsWan
```

Comment:
Should we also add `WAN2` as that port can also be used?

Reply:
Sure